frontend/src/components/phones/dashboard/PhoneDashboard.tsx

```diff
@@ -8,7 +8,7 @@ import {
   ForwardIcon,
   BlockIcon,
   ChevronRightIcon,
-} from "../../../components/Icons";
+} from "../../Icons";
```

Comment:
I renamed this file because it kept tripping me up that the component and CSS module were named `PhoneDashboard` but the file wasn't. Looks like VSCode also fixed the redundant path in here while it was at it.